src/utils/simple_logger.h

```diff
@@ -71,22 +72,22 @@ class simple_logger : public logging_provider
 {
 public:
     simple_logger(const char *log_dir);
-    virtual ~simple_logger(void);
+    ~simple_logger(void);
```

Comment:
Should we mark `override` for those destructors whose ancestors are declared as `virtual` ?

Reply:
I think it would be better to mark `override`.
https://github.com/isocpp/CppCoreGuidelines/issues/721